jupyter/swan.py

```diff
@@ -742,20 +795,17 @@ def simple_df(self):
              # 'percentile/5th',
              # 'percentile/90th',
              # 'percentile/95th',
-             'percentile/99th',
+             PERCENTILE99TH_LABEL,
              ACHIEVED_LATENCY_LABEL,
-             'qps',
+             QPS_LABEL,
              ACHIEVED_QPS_LABEL,
 
-             'llc_occupancy/hp/bytes',
-             'llc_occupancy/hp/perecentage',
 
-             'llc_occupancy/be/bytes',
-             'llc_occupancy/be/perecentage',
+        ]
+        if LLC_HP_LABEL in self.df:
+            columns += extra_columns
 
-             'memory_bandwidth/local/hp/bytes',
-             'memory_bandwidth/local/be/bytes',
-        ]]
+        df = self.df[columns]
```

Comment:
@ppalucki - provide docs (link to cheat sheet preferably) and/or rename the function (eg. to `filter_dataframe` or `shrink_dataframe`) please.